docs/guides/using_ocio/compatible_software.rst

```diff
@@ -62,7 +62,7 @@ C++
 
 The core OpenColorIO API is available for use in C++. Minimal code examples are also available in the source code distribution. 
 
-Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/export/OpenColorIO>`__
+Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/OpenColorIO>`__
```

Comment:
```suggestion
Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/include/OpenColorIO>`__
```
`export` was replaced with `include` in the new repo structure.

Reply:
Thanks! I've made the fix.